src/main/resources/db/migration/V1.3__updateVisit.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE visit
+    ADD COLUMN visit_date_time TIMESTAMP NOT NULL,
+    ADD COLUMN paid VARCHAR(3) NOT NULL,
+    ADD COLUMN description VARCHAR(100) NOT NULL,
```

Comment:
dla mnie jest oki aczkolwiek dwa pytania:
1. czy 100 znaków to wystarczająco?
2. czym się różno VARCHAR od NVARCHAR i dlaczego o to pytam??

Reply:
zwiększyłam do 500, poczytałam i z tego co rozumiem postresql nie ma tego typu, varchar ma w sobie już te specyfikę

Reply:
zrobiłaś test na Polskie znaki??

Reply:
tak działają